homeassistant/components/neato/config_flow.py

```diff
@@ -23,7 +26,7 @@ def logger(self) -> logging.Logger:
         """Return logger."""
         return logging.getLogger(__name__)
 
-    async def async_step_user(self, user_input: dict | None = None) -> dict:
+    async def async_step_user(self, user_input: dict | None = None) -> FlowResult:
```

Comment:
Please type the dict items as far as possible.
```suggestion
    async def async_step_user(self, user_input: dict[str, Any] | None = None) -> FlowResult:
```

Reply:
Thx for spotting, during rebase I missed it ;-(